### PR TITLE
TGUI Drag Inventory

### PR DESCRIPTION
**Pull request by ShadowLarkens**

### CODE REVIEW

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -371,18 +322,14 @@
...
-
-	if (href_list["mach_close"])
+	if (href_list["mach_close"]) // This is horrible.
 		var/t1 = text("window=[]", href_list["mach_close"])
 		unset_machine()
 		src << browse(null, t1)
 
 	if(href_list["item"])
-		handle_strip(href_list["item"],usr)
+		log_runtime(EXCEPTION("Warning: human/Topic was called with item [href_list["item"]], but the item Topic is deprecated!"))
```
Comment (ShadowLarkens) [inline code/modules/mob/living/carbon/human/human.dm:331]: I didn't see anything that could call this, but I'm paranoid of magic bullshit with Topic() calls after seeing some horrible, horrible things during the last few TGUI ports, so fuck it, runtimes.

File: code/modules/mob/living/inventory.dm [hunk 2]
```diff
@@ -180,3 +162,191 @@
...
+	data["internals"] = host.internals
+	data["internalsValid"] = istype(host.wear_mask, /obj/item/clothing/mask) && istype(host.back, /obj/item/weapon/tank)
+
+	return data
+
+/datum/inventory_panel/tgui_act(action, list/params, datum/tgui/ui, datum/tgui_state/state)
+	if(..())
+		return TRUE
+
+	// If anyone wants the inventory panel to actually work,
```
Comment (ShadowLarkens) [inline code/modules/mob/living/inventory.dm:268]: I'm not really sure why non-human living animals even have an inventory panel. They don't have any corresponding Topic() to handle stripping or putting stuff on them. I guess it's just to view their slots?

**Comment by KasparoVy** [comment]
Lmfao that Tajaran random gen name. Love it

**Comment by ShadowLarkens** [comment]
It is pretty great x3

**Pull request merged**